Source/Scene/PointCloud3DTileContent.js

```diff
@@ -732,20 +852,15 @@ define([
         }
     }
 
-    function getGLSLType(type) {
+    function getGlslType(type) {
```

Comment:
In my opinion this is a bad change, because now you can't represent a lot of numbers with it. Numbers in JS are always stored as [double and not just as float](https://www.w3schools.com/js/js_numbers.asp). Another thing: I didn't worked a while with GLSL, but did't I got compiler error in case I compare a float with e.g. an uint without explicit cast? So imho it would be better to cast on JavaScript side instead of in GLSL.

Reply:
Hey @SunBlack, thanks for checking back in on this.

I think this change is ok for a couple reasons:
* The GLSL transpiler already converts all numbers to floats [here](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Expression.js#L1529-L1536), so there shouldn't be an issue with mismatching types.
* glsl 1.0 doesn't have a double type

It's true that large enough integers can't be represented exactly as float, but I think that's a pretty rare use case... I hope.